appveyor.yml

```diff
@@ -72,6 +72,9 @@ install:
     # Remove cygwin (and therefore the git that comes with it).
     - cmd: rmdir C:\cygwin /s /q
 
+    # Install workaround 64-bit support for VS 2008.
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "27" call vs2008\setup_x64.bat
```

Comment:
Please remove this. The point of this package is it will take its place ultimately.